CMakeLists.txt

```diff
@@ -27,7 +27,15 @@ target_include_directories(oboe
                            PRIVATE src 
                            PUBLIC include)
 
+ # Compile Flags:
+ #     Enable -Werror when building debug config
+ #     Enable -Ofast
 target_compile_options(oboe PRIVATE -std=c++11
                             PRIVATE -Wall
-                            PRIVATE "$<$<CONFIG:DEBUG>:-Werror>") # Only include -Werror when building debug config
+                            PRIVATE "$<$<CONFIG:DEBUG>:-Werror>"
+                            PRIVATE -Ofast)
```

Comment:
Shouldn't this be release=`-Ofast`, debug=`-O0`? 
Also why the multiple `PRIVATE` keywords? Couldn't these all go on one line?

Reply:
add -Ofast for debug, leave default for debug build. Done